src/transformers/modeling_tf_utils.py

```diff
@@ -1226,12 +1222,12 @@ def _convert_head_mask_to_5d(self, head_mask, num_hidden_layers):
         head_mask = tf.cast(head_mask, tf.float32)  # switch to float if need + fp16 compatibility
         return head_mask
 
-    def eager_serving(self, inputs):
```

Comment:
We can't just kill a public method like this, unfortunately. We'll need to have a deprecation cycle and warning that it's going to be removed in 2 versions time

Reply:
Understood! I'll put it back and mark it as deprecated.

Reply:
Agree in general, but is `eager_serving` intended to be used by user ..?

Reply:
@ydshieh Nope, not intended, but it's a public method on all of our TF models, so removing it is a breaking change.